src/main/resources/template/error1.vm

```diff
@@ -0,0 +1 @@
+<h1>NO content to display!!</h1>
```

Comment:
please give some meaningful names to the template files, do not use generic names and numbers